lib/inner-slider.jsx

```diff
@@ -127,8 +128,36 @@ var Slider = React.createClass({
         }
       }
 
-      var prevArrow = <button key={0} ref='previous' type="button" data-role="none" className={classnames(prevClasses)} style={{display: 'block'}} onClick={prevHandler}> Previous</button>;
-      var nextArrow = <button key={1} ref='next' type="button" data-role="none" className={classnames(nextClasses)} style={{display: 'block'}} onClick={nextHandler}>Next</button>;
+      var prevArrowProps = {
+        key: '0',
+        ref: 'previous',
+        'data-role': "none",
+        className: classnames(prevClasses),
+        style: {display: 'block'},
+        onClick: prevHandler
+      }
+
+      if (this.props.prevArrow) {
+        var prevArrow = <this.props.prevArrow {...prevArrowProps} />;
```

Comment:
Here you used the prevArrow and nextArrow terminology correctly.
I think you made a mistake in default-props.js file.
Can you correct it.